index.html

```diff
@@ -635,6 +635,142 @@ <h4>Response</h4>
         </p>
       </section>
     </section>
+
+    <section id="error-response">
+      <h3>Error Response</h3>
+      <p>If a <a>Thing</a> experiences an error when attempting to carry out an operation requested by a
+        <a>Consumer</a>,
```

Comment:
I think this statement is rather vague in the sense that we might end up with different implementations.

see for example https://github.com/w3c/web-thing-protocol/pull/78#pullrequestreview-2920001715

Is it okay if the implementations differ?